### PR TITLE
Indent dump output

### PR DESCRIPTION
**Pull request by zaucker**

### CODE REVIEW

File: lib/Agrammon/Model.pm6 [hunk 1]
```diff
@@ -306,7 +306,8 @@ class Agrammon::Model {
     method dump {
         my Str $output;
         for @!evaluation-order.reverse {
-            $output ~= $_.taxonomy ~ "\n";
+            my $level = ($_.taxonomy ~~ m:global/ '::' /).elems;
+            $output ~= $_.taxonomy.indent(4 * $level) ~ "\n";
```
Comment (jnthn) [inline lib/Agrammon/Model.pm6:310]: `$_.taxonomy` can be spelled `.taxonomy` (in line with the general principle that much of the time, you don't have to explicitly mention `$_`).

File: lib/Agrammon/Model.pm6 [hunk 2]
```diff
@@ -306,7 +306,8 @@ class Agrammon::Model {
     method dump {
         my Str $output;
         for @!evaluation-order.reverse {
-            $output ~= $_.taxonomy ~ "\n";
+            my $level = ($_.taxonomy ~~ m:global/ '::' /).elems;
```
Comment (jnthn) [inline lib/Agrammon/Model.pm6:309]: This works, but `.taxonomy.comb('::').elems` will too (`comb` being the opposite of `split` - you say what you want).

**Review by jnthn — APPROVED** [review]
It'll work, there's an easier way to write it.

File: lib/Agrammon/Model.pm6 (hunk 1 above)
Comment (zaucker) [reply lib/Agrammon/Model.pm6:310]: I stole that from a few lines above without really thinking if the `$_`  is really needed here.

**Review by jnthn — APPROVED** [review]

**Pull request merged**